src/lattice/unit_cell.js

```diff
@@ -0,0 +1,31 @@
+// TODO: refactor and remove the need for UnitCell
+export class UnitCell {
+    constructor(ax, ay, az, bx, by, bz, cx, cy, cz, units) {
+        this.ax = ax;
+        this.ay = ay;
+        this.az = az;
+        this.bx = bx;
+        this.by = by;
+        this.bz = bz;
+        this.cx = cx;
+        this.cy = cy;
+        this.cz = cz;
+        this.units = units;
+    }
+
+    vectorA() {
+        return [this.ax, this.ay, this.az];
+    }
+
+    vectorB() {
+        return [this.bx, this.by, this.bz];
+    }
+
+    vectorC() {
+        return [this.cx, this.cy, this.cz];
+    }
+
+    axes() {
+        return [this.vectorA(), this.vectorB(), this.vectorC()];
+    }
+}
```

Comment:
Looks like the whole file is being added anew - are you sure?

Reply:
This was the approach radik took. There is a TODO to remove this class and a TODO to remove reference to it in the only location it's used. We need to know if it can be safely removed before actually removing it.